modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -318,7 +299,7 @@ output application/java
 
 === Fields To Null
 
-The configurations have a checkbox called Can Clear Fields by Updating Field Value to Null. If checked, all the fields in a request that have a Null value are added to the `fieldsToNull` field and sent to Salesforce.
+The configurations have a checkbox called *Can Clear Fields by Updating Field Value to Null*. If checked, all the fields in a request that have a Null value are added to the `fieldsToNull` field and sent to Salesforce.
```

Comment:
This whole section should be removed(Fields to null) as it does not apply anymore for Salesforce 10.